courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb

```diff
@@ -34,36 +34,10 @@
    "metadata": {},
```

Comment:
<p>Line 1, Replace with this:  Here we will visualize our data using the figure() function in the pyplot module of matplotlib's library -- which is used to create a new figure.</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Updated the same.